Makefile

```diff
@@ -91,9 +91,16 @@ refactor: requires_nix_shell
 		hlint $(HLINT_EXTS) --refactor --refactor-options='-i -s' $$src ;\
 	done
 
+# Compile at source changes
 watch: requires_nix_shell ogmios-datum-cache.cabal
 	while sleep 1; do find ogmios-datum-cache.cabal src test | entr -cd make build; done
 
+run-testnet : requires_nix_shell 
+	./setPrivateNetwork.sh
+
+clean-testnet : requires_nix_shell
+	rm -Rf test-env/ogmios-datum-cache-private-network/*
```

Comment:
I'd be _extremely_ careful with `rm -Rf` and globs

Reply:
yep, I would delete them once I swap the pr from draft.